### DISCUSSION TITLE
Better UI for Volume Control (Phone or Tablet)

### DISCUSSION THREAD

**Discussion by SN-git-stack** [music-assistant/support#2265] [Feature requests and ideas]
When using music assistant I find it difficult to use the volume control, as the slider is rather small.

Possible solutions:
- Add a focused volume control on long press 
- home assistant function for volume control music assistant not the device as this already works

**Comment by harryfine** [comment]
Agree completely. My biggest UI gripe.

**Reply by OzGav** [reply]
If you tap on the speaker icon a larger slider will open. However, I do note that you can only adjust that when the player is ON even though in the slider in the player bar is always adjustable. We will look at that inconsistent behaviour

**Comment by njgco** [comment]
This is also my biggest gripe.   It's usable, but for the entire family, we need to think of a better way to handle the volume UI so it's an easy, 'thoughtless' action.  Right now it takes focus and difficult to make granular adjustments accurately and easily.  It's a core function IMO and should be a priority.

**Comment by OzGav** [comment]
There are now + and - buttons either side of the slider

**Reply by SN-git-stack** [reply]
Thank you. This is a great feature.

**Comment by SN-git-stack** [comment]
Solved: + / - option was added.
